galleries/examples/statistics/customized_violin.py

```diff
@@ -36,20 +36,30 @@ def set_axis_style(ax, labels):
 np.random.seed(19680801)
 data = [sorted(np.random.normal(0, std, 100)) for std in range(1, 5)]
 
-fig, (ax1, ax2) = plt.subplots(nrows=1, ncols=2, figsize=(9, 4), sharey=True)
+fig, (ax1, ax2, ax3) = plt.subplots(nrows=1, ncols=3, figsize=(9, 3), sharey=True)
 
 ax1.set_title('Default violin plot')
 ax1.set_ylabel('Observed values')
 ax1.violinplot(data)
 
-ax2.set_title('Customized violin plot')
-parts = ax2.violinplot(
-        data, showmeans=False, showmedians=False,
-        showextrema=False)
+ax2.set_title('Set colors of violins')
+ax2.set_ylabel('Observed values')
+ax2.violinplot(
+    data,
+    facecolor=[('y', 0.3), ('b', 0.3), ('r', 0.3), ('g', 0.3)],
+    linecolor='k',
+)
+# Note that when passing a sequence of colors, the method will repeat the sequence if
+# less colors are provided than data distributions.
+
+ax3.set_title('Customized violin plot')
+parts = ax3.violinplot(
+        data, showmeans=False, showmedians=False, showextrema=False,
+        facecolor='#D43F3A', linecolor='k')
 
 for pc in parts['bodies']:
-    pc.set_facecolor('#D43F3A')
-    pc.set_edgecolor('black')
+    pc.set_edgecolor('k')
```

Comment:
```suggestion
    pc.set_edgecolor('black')
```
Let's keep the long form because it's easier to read.

Reply:
done